com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -520,6 +520,17 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
             if (useRenderPassEnabled || useDepthPriming)
                 createColorTexture |= createDepthTexture;
 
+            //Set rt descriptors so preview camera's have access should it be needed
+            CommandBuffer cmd = CommandBufferPool.Get();
```

Comment:
The CommandBuffer is not actually used if you follow the calls. Would be best to remove it.

Reply:
Good catch. I removed the CommandBuffer